app/src/main/java/de/westnordost/streetcomplete/data/osm/mapdata/ElementDao.kt

```diff
@@ -78,11 +78,11 @@ class ElementDao(
     }
 }
 
-private data class ElementIds(val nodes: List<Long>, val ways: List<Long>, val relations: List<Long>) {
+data class ElementIds(val nodes: List<Long>, val ways: List<Long>, val relations: List<Long>) {
     val size: Int get() = nodes.size + ways.size + relations.size
 }
 
-private fun Iterable<ElementKey>.toElementIds(): ElementIds {
+fun Iterable<ElementKey>.toElementIds(): ElementIds {
```

Comment:
These are not meant to be public. Do they need to be? I don't find them referenced anywhere